--- sprox-0.11.2/sprox/test/base.py.bak	2020-05-03 22:34:41.000000000 +0200
+++ sprox-0.11.2/sprox/test/base.py	2021-02-12 16:13:12.578595958 +0100
@@ -59,7 +59,7 @@
         new_node.text = ''
     if new_node.tail and new_node.tail.strip() == '':
         new_node.tail = ''
-    for child in node.getchildren():
+    for child in node:
         if child is not None:
             child = remove_whitespace_nodes(child)
         new_node.append(child)
@@ -67,7 +67,7 @@
 
 def remove_namespace(doc):
     """Remove namespace in the passed document in place."""
-    for elem in doc.getiterator():
+    for elem in doc.iter():
         match = re.match('(\{.*\})(.*)', elem.tag)
         if match:
             elem.tag = match.group(2)
